packages/crayons-core/src/components/popover/popover.tsx

```diff
@@ -108,7 +120,12 @@ export class Popover {
   }
 
   @Method()
-  async show() {
+  async delayPopOverTrigger(fn, delay) {
```

Comment:
Can you please elaborate the reason why `delayPopOverTrigger` is exposed as a public method ? , we usually decorate a function with method decorator if it needs to be exposed as a public API.

I think, we can just add one more method in addition to existing `show` method , with name like `showWithDelay` that can accept delay as a param and that can internally call `show` within setTimeout. Similarly for `hide`.

Reply:
Agree, I was not aware of the public API exposing part - Corrected it. 

Regarding the show/hide delay logic, didn't expose any public methods and tried to handle trigger with a handler method to avoid duplication of main logic.

Correct me if this as well can be improved.

Reply:
Thanks for the implementing the changes!